Add documentation for removing agent elevated privileges (#4910)
elastic/docs-content: reference/fleet/remove-agent-elevated-privileges.md

:::

:::{note}
Running without root privileges is not supported for {{agents}} installed using RPM or DEB packages.
:::

## Requirements [remove-elevated-privileges-requirements]

To be eligible for elevated privilege removal, {{agents}} must meet the following requirements:

* The agent must be running {{agent}} version 9.3.0 or later.
* The agent must not be a {{fleet-server}} agent.
* The agent must not be assigned to an agent policy that contains integrations requiring elevated privileges.

## Remove elevated privileges using the {{fleet}} UI [remove-elevated-privileges-ui]

To remove elevated privileges from one or more {{agents}} using the {{fleet}} UI:

1. In {{kib}}, find **Fleet** in the navigation menu or use the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md).

2. Select one or more agents for which you want to remove the elevated privileges.

3. Select **Actions** > **Security and removal** > **Remove root access for N agents** where N is the number of agents you've selected. A flyout opens.

4. Review the information in the flyout:

>    * A warning is displayed if any of the selected agents do not meet the requirements for privilege removal. To view a list of these agents, click **View hosts**.

Reviewer: ```suggestion
   * A warning is displayed if any of the selected agents do not meet the requirements for privilege removal. To view a list of these agents, select **View hosts**.
```

      - Username
      - User group
      - User password

      If the user details are not specified, {{agent}} uses the default unprivileged user (`elastic-agent-user`) or creates it if it doesn't exist.

      ::::{note}
      For agents running on Windows, the password is required when specifying a custom user account.
      ::::

6. To confirm, select **Remove privilege for N agents** where N is the number of agents eligible for privilege removal.

The eligible agents are switched to unprivileged mode. You can monitor the progress on the **Agents** tab. After the operation completes, the agents run as unprivileged users and their privilege mode is updated in the agent details.

:::{note}
You can also initiate the privilege removal for a single agent from either of these locations:

- The action menu at the end of the agent's row on the **Agents** tab
- The **Actions** menu on the agent's **Agent details** page

From either location, select **Security and removal** > **Remove root privilege**, then review the information in the flyout and confirm to remove privileges.
:::

> ## Remove elevated privileges using the {{fleet}} API [remove-elevated-privileges-api]

Author: I don’t think it’s worth separating the info on removing the elevated privileges via the API into a separate doc. It’s a bit of an “extra info” for advanced users, but I don’t want to recreate the API docs here, just add info that it’s possible and add a basic example. 

I’ll move this section to after “What happens.."

> You can also remove elevated privileges using the {{fleet}} API. This is useful for automation or when managing a large number of agents.

Reviewer: ```suggestion
You can also remove elevated privileges using the {{fleet}} API. This is useful for automation or when managing multiple agents.
```
